orca-clouddriver/src/test/groovy/com/netflix/spinnaker/orca/clouddriver/tasks/monitoreddeploy/EvaluateDeploymentHealthTaskSpec.groovy

```diff
@@ -198,6 +204,200 @@ class EvaluateDeploymentHealthTaskSpec extends Specification {
     false              | null             || ExecutionStatus.FAILED_CONTINUE
   }
 
+  def "should handle retrofit http error and return the task status depending on the scenarios"() {
+  def converter = new JacksonConverter(new ObjectMapper())
+
```

Comment:
not a major concern but looks like the indentation is not proper.

Reply:
Corrected the indentation